app/src/main/java/org/wikipedia/theme/ThemeFittingRoomFragment.kt

```diff
@@ -0,0 +1,65 @@
+package org.wikipedia.theme
+
+import android.graphics.Typeface
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import io.reactivex.rxjava3.disposables.CompositeDisposable
+import io.reactivex.rxjava3.functions.Consumer
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.databinding.FragmentThemeFittingRoomBinding
+import org.wikipedia.events.ChangeTextSizeEvent
+import org.wikipedia.events.WebViewInvalidateEvent
+import org.wikipedia.settings.Prefs
+
+class ThemeFittingRoomFragment : Fragment() {
+    private val disposables = CompositeDisposable()
+    private var _binding: FragmentThemeFittingRoomBinding? = null
+    private val binding get() = _binding!!
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
```

Comment:
Very minor but I would recommend that put `binding` and `_binding` on the top of these variables.

and please add an empty line between line `21` and line `22`.

Reply:
Done @cooltey